### DISCUSSION TITLE
When the second client connect to server, it will failed...

### DISCUSSION THREAD

**Discussion by jason8745** [bluenviron/mediamtx#748] [Questions]
Hello @aler9, this is a very helpful application!  I have some trouble when using it . In order to reduce the pressure of our ip camera(becauser the numbers of rtsp links is decided by the cpu of camera ), we use this rtsp-server as a proxy . Our applications will connect to the rtsp server instead of ip camera directly. In our plan, one rtsp source need to be read by many applications. But I found that when the second application get the rtsp that been used . It will connected for a while, and then closed(EOF)... 
 
`2021/12/23 08:37:53 INF [RTSP] [conn xx.xx.xx.xx:xx] closed (EOF)
2021/12/23 08:37:53 INF [RTSP] [session 868309782] closed (teared down by xx.xx.xx.xx:xx)`

version: v0.17.12
I used both VLC and opencv-python and both of them can connected for a while, and then closed(EOF).
Thanks for your help!

**Comment by jason8745** [comment]
> Hello @aler9, this is a very helpful application! I have some trouble when using it . In order to reduce the pressure of our ip camera(becauser the numbers of rtsp links is decided by the cpu of camera ), we use this rtsp-server as a proxy . Our applications will connect to the rtsp server instead of ip camera directly. In our plan, one rtsp source need to be read by many applications. But I found that when the second application get the rtsp that been used . It will connected for a while, and then closed(EOF)...
> 
> `2021/12/23 08:37:53 INF [RTSP] [conn xx.xx.xx.xx:xx] closed (EOF) 2021/12/23 08:37:53 INF [RTSP] [session 868309782] closed (teared down by xx.xx.xx.xx:xx)`
> 
> version: v0.17.12 I used both VLC and opencv-python and both of them can connected for a while, and then closed(EOF). Thanks for your help!

This the config that I use for one ip camera
![螢幕擷取畫面 2021-12-23 164959](https://user-images.githubusercontent.com/41944427/147214290-707193d8-1925-4f8a-a463-e68651735eb5.png)

**Reply by aler9** [reply]
Hello, "connection teared down by" means that one of your clients (VLC or opencv) closed the connection voluntarily. You can understand the reason by looking at their logs. The server didn't force them to disconnect. At the same time, the server supports an unlimited number of readers, therefore reading the same stream with two clients is fully supported.

**Reply by neilyoung** [reply]
I would suggest to re-phrase it to "torn down", which is IMHO the proper phrase

**Reply by aler9** [reply]
@neilyoung you're perfectly right, i updated the error message.

[a comment by an automated account is not shown]
